css/mistica-common.css

```diff
@@ -939,6 +939,114 @@ button.mistica-display-card:has(.mistica-card__media):active:after {
     height: auto;
 }
 
+/* Grid Layout */
+
+.mistica-grid-layout {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: 24px;
+}
+
+@media (max-width: 1023px) {
+  .mistica-grid-layout {
+    grid-template-columns: minmax(0, 1fr);
+  }
+
+  .mistica-grid-layout [class^="mistica-col-"] {
+    grid-column: span 1;
+  }
+
+  .mistica-col-1 {
+    display: none;
+  }
+
+  .mistica-vertical-spacing-0 {
+    gap: 0;
+  }
```

Comment:
I'd call these classes `.mistica-grid-vertical-space-0` and use `row-gap` instead of `gap`